packages/builder/src/builtin-tasks/compile.ts

```diff
@@ -0,0 +1,72 @@
+import { task } from "../internal/core/config/config-env";
+import { networks } from "../../sample-project/algob.config";
+import { TASK_COMPILE } from "./task-names";
+import YAML from "yaml";
+const algosdk = require('algosdk');
+const fs = require('fs');
+const path = require('path');
+const murmurhash = require('murmurhash');
+
+/* Credentials */
+const token = networks.localhost.token;
+const host = networks.localhost.host;
+const port = networks.localhost.port;
+
+let algodclient = new algosdk.Algodv2(token, host, port);
+
+export default function (): void {
+  task(TASK_COMPILE, "Compilation task")
+    .setAction(
+      async () => {
```

Comment:
Create a new method before the call to `setAction` and name it. Only call it here.